testing/py-geoip-legacy/APKBUILD

```diff
@@ -0,0 +1,53 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-geoip-legacy
```

Comment:
Why have you named it like this when the upstream name is geoip-api-python (so py-geoip-api)?